sensor/common/sensor/sensor.go

```diff
@@ -318,13 +317,9 @@ func (s *Sensor) waitUntilCentralIsReady() {
 func (s *Sensor) pollMetadata() error {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
-	md, err := s.centralRestClient.GetMetadata(ctx)
-
-	if err != nil {
-		return err
-	}
-	if md.GetLicenseStatus() != v1.Metadata_VALID {
-		return errors.Errorf("central license status is not VALID but %v", md.GetLicenseStatus())
+	// Metadata result doesn't matter, as long as central is reachable
+	if _, err := s.centralRestClient.GetMetadata(ctx); err != nil {
+		return errors.Wrap(err, "polling metadata")
```

Comment:
I'm not sure the `if` and `Wrap` are adding any value here, "polling metadata" is also not likely to help someone who does not already know what sensor is doing. Perhaps just `return err`?

Reply:
Done